crates/colorchoice-clap/src/lib.rs

```diff
@@ -36,66 +38,6 @@ impl Color {
     }
 }
 
-/// Argument value for when to color output
-#[derive(Copy, Clone, Debug, PartialEq, Eq)]
-pub enum ColorChoice {
-    Auto,
-    Always,
-    Never,
-}
-
-impl ColorChoice {
-    /// Report all `possible_values`
-    pub fn possible_values() -> impl Iterator<Item = clap::builder::PossibleValue> {
-        use clap::ValueEnum;
-        Self::value_variants()
-            .iter()
-            .filter_map(clap::ValueEnum::to_possible_value)
-    }
-
-    fn as_str(&self) -> &'static str {
-        match self {
-            Self::Auto => AUTO,
-            Self::Always => ALWAYS,
-            Self::Never => NEVER,
-        }
-    }
-}
-
-impl Default for ColorChoice {
-    fn default() -> Self {
-        Self::Auto
-    }
-}
-
-impl std::fmt::Display for ColorChoice {
```

Comment:
The commit describes this as a refactor and it and the PR don't describe any breaking changes.  Can you verify and explicitly state that there **aren't** breaking changes?  A change like this is putting more of the burden on that to the reviewer when the contribution should be more explicit.

(and for me to be explicit, I do not think this change justifies a major release so please maintain compatibility)

Reply:
I thought I checked that it doesn't break anything, but it looks like it removes `ColorChoice::possible_values` fn. 

> and for me to be explicit, I do not think this change justifies a major release so please maintain compatibility

I do agree with that, but if we want to add the `possible_values` fn back, I don't think we can re-use the `clap::ColorChoice` here since it's an enum.

The only solution I can think of is adding that function in `clap`.

Reply:
I think it could be reasonable to add that function.  If so, we should look into doing similar for other `ValueEnum`s (`clap_complete`s `Shell`)